common/src/main/java/com/railwayteam/railways/compat/tracks/GenericTrackCompat.java

```diff
@@ -31,10 +32,14 @@
 import static com.simibubi.create.content.trains.track.TrackMaterialFactory.make;
 
 public class GenericTrackCompat {
+    public final Mods mod;
     public final String modid;
+    public static boolean modLoaded = false;
```

Comment:
this doesn't work as static, there will not be a separate instance of the static variable for every child-class.
Please change it to a non-static variable and reference it in `registerTracksAnyway` instead.